helm-charts/konk/templates/issuer.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: Issuer
+metadata:
+  name: {{ include "konk.fullname" . }}-ca-issuer
+  namespace: {{ .Release.Namespace }}
```

Comment:
Include the labels like in the other resources: https://github.com/infobloxopen/konk/blob/cf35713a340b9967d56f179319da2b10a6af6570/helm-charts/konk/templates/configmap.yaml#L6-L7